src/components/WalletModal/PendingView.tsx

```diff
@@ -82,20 +83,22 @@ export default function PendingView({
         <LoadingWrapper>
           {error ? (
             <ErrorGroup>
-              <div>Error connecting.</div>
+              <div>
+                <Trans id="error.connecting">Error connecting</Trans>
```

Comment:
missing dot ".'

Reply:
intentional for key consistency, thanks for flagging it!